### PR TITLE
Added Microsoft Edge support

### PR DESCRIPTION
**Pull request by Shine-neko**

### CODE REVIEW

File: lib/Ikimea/Browser/Browser.php [hunk 1]
```diff
@@ -564,6 +566,26 @@ protected function checkBrowserInternetExplorer()
...
+    protected function checkBrowserEdge()
+    {
+        if (stripos($this->_agent, 'edge') !== false) {
+            $aresult = explode('/', stristr($this->_agent, 'Edge'));
+            $aversion = explode(' ', $aresult[1]);
+            $this->setVersion($aversion[0]);
+            $this->setBrowser(self::BROWSER_EDGE);
+
+            return true;
+
```
Comment (Nek-) [inline lib/Ikimea/Browser/Browser.php]: useless line

Comment (Shine-neko) [inline lib/Ikimea/Browser/Browser.php]: There is a condition that causes this. maybe again later :D

**Comment by Nek-** [comment]
Cool but fails :p

**Pull request merged**